custom/opm/reports.py

```diff
@@ -1043,7 +1043,7 @@ def headers(self):
 
     def get_matching_awcs(self):
         filter_on = 'awc' if self.awcs else 'gp' if self.gp else 'block'
-        for block, gps in get_hierarchy().items():
+        for block, gps in user_data_as_hierarchy().items():
```

Comment:
Actually all of this function is not even needed. Since the resulting case_objects are already filtered based on selected report filters. I will remove all of this.